server/routes.js

```diff
@@ -107,7 +107,7 @@ router.get('/authenticated', function(req, res) {
  * the lack of JWTs.
  */
 router.get('/authorized/:profileId', function(req, res) {
-  if (req.user.id === req.params.profileId) {
+  if (req.user && req.user.id === req.params.profileId) {
```

Comment:
I add this in my pull request but there should be a check to see if the req.user exists before checking the .id